docs/en/engines/table-engines/integrations/azure-queue.md

```diff
@@ -50,9 +50,9 @@ To get a list of settings, configured for the table, use `system.azure_queue_set
 
 `SELECT` is not particularly useful for streaming import (except for debugging), because each file can be imported only once. It is more practical to create real-time threads using [materialized views](../../../sql-reference/statements/create/view.md). To do this:
 
-1.  Use the engine to create a table for consuming from specified path in S3 and consider it a data stream.
-2.  Create a table with the desired structure.
-3.  Create a materialized view that converts data from the engine and puts it into a previously created table.
+1.Use the engine to create a table for consuming from specified path in S3 and consider it a data stream.
+2.Create a table with the desired structure.
+3.Create a materialized view that converts data from the engine and puts it into a previously created table.
```

Comment:
Same here. Please make sure these changes don't happen, as the list is no longer rendered without the space.